packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -0,0 +1,299 @@
+import {LightningElement, api, track, wire} from 'lwc';
+import caseClassificationTitle from '@salesforce/label/c.quantic_CaseClassificationTitle';
+import moreTopics from '@salesforce/label/c.quantic_MoreTopics';
+import selectOption from '@salesforce/label/c.quantic_SelectOption';
+import {
+  getObjectInfo,
+  getPicklistValuesByRecordType,
+} from 'lightning/uiObjectInfoApi';
+import CASE_OBJECT from '@salesforce/schema/Case';
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+} from 'c/quanticHeadlessLoader';
+
+/** @typedef {import("coveo").CaseAssistEngine} CaseAssistEngine */
+/** @typedef {import("coveo").CaseField} CaseField */
+
+/**
+ * The `QuanticCaseClassification` component displays field value suggestion returned by Coveo Case Assist as well as a single-select dropdown containing other available values.
+ *
+ * @category Case Assist
+ * @example
+ * <c-quantic-case-classification engine-id={engineId} field-name="Priority" required label="Which topic is related to your issue?" select-placeholder="More topics" max-choices="4" message-when-value-missing="Select an option"></c-quantic-case-classification>
+ */
+export default class QuanticCaseClassification extends LightningElement {
+  @wire(getObjectInfo, {objectApiName: CASE_OBJECT})
+  objectInfo;
+
+  @wire(getPicklistValuesByRecordType, {
+    recordTypeId: '$objectInfo.data.defaultRecordTypeId',
+    objectApiName: CASE_OBJECT,
+  })
+  picklistValues;
+
+  labels = {
+    caseClassificationTitle,
+    moreTopics,
+    selectOption,
+  };
+
+  /**
+   * The ID of the engine instance the component registers to.
+   * @api
+   * @type {string}
+   */
+  @api engineId;
+
+  /**
+   * The field of the case to be classified.
```

Comment:
You will have to be precise here if the fieldName is the salesforce API name, a label, or the Coveo field name